bin/src/data/csv.py

```diff
@@ -180,7 +180,7 @@ def shuffle_labels(self) -> None:
         """
         label_keys = self.get_keys_based_on_name_category_dtype(category='label')
         for key in label_keys:
-            self.data = self.data.with_columns(pl.Series(key, np.random.permutation(self.data[key])))
+            self.data = self.data.with_columns(pl.Series(key, np.random.permutation(list(self.data[key]))))
```

Comment:
why does it have to become a list?

Reply:
because otherwise np.random.permutation does not work on the data frame Series type of data